keyboards/helix/rev2/keymaps/froggy_106/keymap.c

```diff
@@ -0,0 +1,901 @@
+#include QMK_KEYBOARD_H
+#include "bootloader.h"
+#include "keymap_jp.h"
+#ifdef PROTOCOL_LUFA
+#include "lufa.h"
+#include "split_util.h"
+#endif
+#ifdef AUDIO_ENABLE
+  #include "audio.h"
+#endif
+#ifdef SSD1306OLED
+  #include "ssd1306.h"
+#endif
+
+extern keymap_config_t keymap_config;
+
+#ifdef RGBLIGHT_ENABLE
+//Following line allows macro to read current RGB settings
+extern rgblight_config_t rgblight_config;
+#endif
```

Comment:
This shouldn't be needed, as there are a number of query functions.
https://docs.qmk.fm/#/feature_rgblight?id=query

Reply:
Removed